frontend/bank-transfer-ui/src/styles.css

```diff
@@ -10,9 +10,49 @@
   font-style: normal;
 }
 
-body {
-  min-height: 100vh !important;
+html, body {
+  height: 100% !important;
+}
+.wrapper {
```

Comment:
Ten wrapper też raczej jest zbędny, bootstrap 4 ma takie klasy, które mogą Ci to zastąpić